jina/helper.py

```diff
@@ -1237,15 +1237,8 @@ def _get_ip(url):
         'https://checkip.amazonaws.com/',
     ]
 
-    threads = []
-
     for idx, ip in enumerate(ip_server_list):
-        t = threading.Thread(target=_get_ip, args=(ip,))
-        threads.append(t)
-        t.start()
-
-    for t in threads:
-        t.join(timeout)
+        results.append(_get_ip(ip))
```

Comment:
```suggestion
        r = _get_ip(ip)
        if r:
            return r
```

Now that it is not async we can just return the first one that is not None to save time. (As we only return one IP later anyway)

Reply:
Good